src/components/ProgressBar/index.js

```diff
@@ -0,0 +1,5 @@
+import ProgressBar from './ProgressBar'
+
+export { ProgressBar }
```

Comment:
Você pode resumir as 3 linhas em uma só :D 
```suggestion
export { default } from './ProgressBar'
```